docs/customization/configuration.md

```diff
@@ -78,6 +78,7 @@ values via a `docker-compose.override.yml` or `.env` file. You can copy
  set this to a highly random string in case that changes in the future.
 - `ONI_SOLR_URL` (default = `http://solr:8983`): Solr server base URL
 - `ONI_STORAGE_PATH` (default = `(ONI base dir path)/data`): Path to batch storage
+- `SOLR_CLOUD` (default = False): Set to True if running solr in cloud mode
```

Comment:
Can you change this to match the ENV VAR name `ONI_SOLR_CLOUD` rather than the name of the variable within the settings file? This is under the Environment Variables header so that's which name fits best here.